src/dlgButtonSelect.cpp

```diff
@@ -0,0 +1,233 @@
+/***************************************************************************
+ *   Copyright (C) 2008-2013 by Heiko Koehn - KoehnHeiko@googlemail.com    *
+ *   Copyright (C) 2014 by Ahmed Charles - acharles@outlook.com            *
+ *   Copyright (C) 2016 by Stephen Lyons - slysven@virginmedia.com         *
+ *   Copyright (C) 2017 by Ian Adkins - ieadkins@gmail.com                 *
```

Comment:
Hang on, this code is all yours @dicene, no-one else has :copyright: (or blame) for this... :stuck_out_tongue_winking_eye: